doc/data/messages/u/use-dict-literal/details.rst

```diff
@@ -0,0 +1 @@
+Use a dict literal instead of keywords and `dict()`. The literal is faster as it avoids an additional function call.

```

Comment:
Look like this is already in the description of the message (https://github.com/PyCQA/pylint/pull/7691/files#diff-af1d215b95c8b3ac59218ba588186da4aefd936eef4732518196ea309c3cc788R469) so we could remove the details. We could point to a benchmark of the perf different however.

Reply:
https://gist.github.com/hofrob/ad143aaa84c096f42489c2520a3875f9
I tried this simple benchmark script on my machine. Not sure if this counts as a source :grin:.

Reply:
> Not sure if this counts as a source

Yes 👍

Reply:
As discussed, I added the python version and removed the exact percentage.